kubernetes/charts/edge-kubernetes/values.yaml

```diff
@@ -85,9 +85,13 @@ edgeAgent:
 
 # Optional registry credentials for module images
 # registryCredentials:
-#   <SERVER ADDRESS>:
-#     username: '<USERNAME>'
-#     password: '<PASSWORD>'
+#   serveraddress: <SERVER ADDRESS>:
```

Comment:
This change seems suspicious.
use to be
```
registryCredentials: 
  mcr.io: 
    username: user
    password: pass
```
now
```
registryCredentials: 
  serveraddress: mcr.io
  username: user
  password: pass
```

Reply:
You are right! Thanks for pointing out that we are trying to consolidate our registry creds. I will revert this now since we already have a task to tackle reg creds for iotedged-proxy as well.